conformance/tests/httproute-invalid-cross-namespace-backend-ref.go

```diff
@@ -71,5 +72,17 @@ var HTTPRouteInvalidCrossNamespaceBackendRef = suite.ConformanceTest{
 		// Listener ResolvedRefs RefNotPermitted condition once
 		// https://github.com/kubernetes-sigs/gateway-api/issues/1112
 		// has been resolved
+
+		gwAddr := kubernetes.GatewayAndHTTPRoutesMustBeReady(t, suite.Client, suite.ControllerName, kubernetes.NewGatewayRef(gwNN))
```

Comment:
```suggestion
		gwAddr := kubernetes.GatewayAndHTTPRoutesMustBeReady(t, suite.Client, suite.ControllerName, kubernetes.NewGatewayRef(gwNN), routeNN)
```
I believe this PR resolves the TODO on line 49 - if we're expecting a 500 response, the route should be `Accepted` by the gateway because _something_ is being programmed in the underlying proxy for this route.

Reply:
Yeah, I agree.